src/internal/lib/connectors/services/water/BillingBatchService.js

```diff
@@ -61,8 +61,8 @@ class BillingBatchService extends ServiceClient {
     return this.serviceRequest.post(uri, options);
   }
 
-  deleteInvoiceFromBatch (batchId, invoiceId) {
-    const uri = this.joinUrl(`/billing/batches/${batchId}/invoices/${invoiceId}`);
+  deleteInvoiceFromBatch (batchId, invoiceId, originalInvoiceId, rebillInvoiceId) {
+    const uri = this.joinUrl(`/billing/batches/${batchId}/invoices/${invoiceId}/org/${originalInvoiceId}/rebill/${rebillInvoiceId}`);
```

Comment:
this delete route is also used for deleting normal invoices that are not rebills therefore you can drop the last part /org/${originalInvoiceId}/rebill/${rebillInvoiceId} and add these as query params. They are optional in the water service so therefore only tag them on as query params when needed. 

You will need to add a check here if originalInvoiceId and rebillInvoiceId is not empty then add them as query params